user_tools/docs/qualx.md

```diff
@@ -9,20 +9,20 @@ Tool to qualify Spark applications for GPU acceleration, based on XGBoost.
 To predict the speedup of running a Spark application with Spark RAPIDS on GPUs, use the cmd below. It uses an XGBoost model trained on matching CPU and GPU runs of various Spark applications.
 
 ```bash
-spark_rapids prediction --qual_output </path/to/qual_output> --prof_output </path/to/prof_output> --output_folder </path/to/save/csv/files>
+spark_rapids prediction \
+--qual_output </path/to/qual_output> \
+--prof_output </path/to/prof_output> \
+--output_folder </path/to/save/csv/files>
 ```
 
-#### Arguments:
+Arguments:
 - qual_output: Path to a directory containing qualification tool output.
 - prof_output: Path to a directory containing profiling tool output.
 - output_folder: Path to store the output.
 
 ### Training
 
-To train an XGBoost model on the specific dataset, follow these steps below. Refer to [Getting Started](../README.md#getting-started) section for installing the required dependencies for training.
-
-#### Data Preparation
-TODO
+To train an XGBoost model on a specific dataset, follow these steps below. Refer to [Getting Started](../README.md#getting-started) section for installing the required dependencies for training.
```

Comment:
use absolute link instead of relative because this will be broken if referenced from another site.

```
[Getting Started](https://github.com/NVIDIA/spark-rapids-tools/blob/main/user_tools/docs/README.md#getting-started)
```
```